README.md

```diff
@@ -6,19 +6,136 @@ The web interface is under development right now. You can try it at [ichi.moe](h
 2. Create a settings.lisp file based on the provided settings.lisp.template file with the correct paths to the abovementioned files and the database connection parameters.
-3. The code can be loaded as a regular ASDF system. Use quicklisp to easily install all the dependencies. 
-4.
-   * Easy mode: Use database dump from [the release page](https://github.com/tshatrov/ichiran/releases) to create a suitable database. Make sure ```settings.lisp``` contains the correct connection parameters. Use ```(ichiran/maintenance:add-errata)``` to make database up to date.
-   * Hard mode: Use ```(ichiran/maintenance:full-init)``` to completely initialize the database. Use ```(ichiran/maintenance:load-jmdict)``` followed by ```(ichiran/maintenance:load-best-readings)``` to initialize only `ichiran/dict` and not `ichiran/kanji`. Either way, this will take a few hours or so.
-5. Use ```(ichiran/test:run-all-tests)``` to check that the installation satisfies the tests.
-6. Before using any word segmenting functionality, run ```(ichiran/dict:init-suffixes t)``` to create a suffix cache, which will improve the quality of segmentation.
+3. The code can be loaded as a regular ASDF system. Use quicklisp to easily install all the dependencies.
+4. - Easy mode: Use database dump from [the release page](https://github.com/tshatrov/ichiran/releases) to create a suitable database. Make sure `settings.lisp` contains the correct connection parameters. Use `(ichiran/maintenance:add-errata)` to make database up to date.
+   - Hard mode: Use `(ichiran/maintenance:full-init)` to completely initialize the database. Use `(ichiran/maintenance:load-jmdict)` followed by `(ichiran/maintenance:load-best-readings)` to initialize only `ichiran/dict` and not `ichiran/kanji`. Either way, this will take a few hours or so.
+5. Use `(ichiran/test:run-all-tests)` to check that the installation satisfies the tests.
+6. Before using any word segmenting functionality, run `(ichiran/dict:init-suffixes t)` to create a suffix cache, which will improve the quality of segmentation.
+
+## Dockerized version
+
+Build (executed from the root of this repo):
+
+```
+docker compose build
+```
+
+Start containers (this will take longer for the first time, because the db will get imported from the dump here, and other ichiran initializations will also get done here):
+
+```
+docker compose up
+```
+
+If there were errors while importing db, or you want to import a new database you need to delete postgres data, so the postgres docker initdb scripts get called (if the folder is not empty it won't get called), and after this you can call `docker compose up` again:
+
+```
+sudo rm -rf docker/pgdata
+```
+
+Test suite:
+
+```
+$ docker exec -it ichiran-main-1 test-suite
```

Comment:
```
sudo docker ps
CONTAINER ID   IMAGE          COMMAND                  CREATED          STATUS          PORTS      NAMES
0c3854eb0d84   ichiran_pg     "docker-entrypoint.s…"   47 minutes ago   Up 47 minutes   5432/tcp   ichiran_pg_1
3689e0ad193e   ichiran_main   "/bin/sh -c entrypoi…"   47 minutes ago   Up 47 minutes              ichiran_main_1
```

At least for me the container is called `ichiran_main_1` not `ichiran-main-1`. Not sure if this is system specific or you had renamed it at some point.

Reply:
Interesting, must be a version difference, I checked my bash history and I definitely used names with `-`. I'll update the README this weekend if I have time.